Library/Formula/kafka.rb

```diff
@@ -14,7 +14,7 @@ class Kafka < Formula
 
   depends_on "gradle"
   depends_on "zookeeper"
-  depends_on :java => "1.7+"
+  depends_on :java => "1.7"
```

Comment:
My attempt at writing a pour_bottle method is not working like it should.
  def pour_bottle?
    system("/usr/libexec/java_home --version {A VERSION I DON'T HAVE INSTALLED} --failfast")
  end

When I run brew install --verbose --debug kafka instead of doing what I hoped, brew just quits the install printing out
==> Formula
Path: /usr/local/Library/Formula/kafka.rb
==> Configuration
HOMEBREW_VERSION: 0.9.5
ORIGIN: https://github.com/Homebrew/homebrew
HEAD: 2f214881a76106e939fb87f1e3c5efded5ea2533
Last commit: 2 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CELLAR: /usr/local/Cellar
CPU: quad-core 64-bit ivybridge
OS X: 10.10.2-x86_64
Xcode: 6.2
CLT: 6.2.0.0.1.1424975374
Clang: 6.0 build 600
X11: 2.7.7 => /opt/X11
System Ruby: 2.0.0-p481
Perl: /usr/bin/perl
Python: /usr/bin/python
Ruby: /Users/derek.bassett/.rbenv/shims/ruby => /Users/derek.bassett/.rbenv/versions/2.1.3/bin/ruby
Java: 1.8.0_40
==> ENV

with no explanation.